packages/gafl-webapp-service/src/handlers/analytics-handler.js

```diff
@@ -29,7 +29,11 @@ export default async (request, h) => {
   const { payload } = request
   const analytics = await request.cache().helpers.analytics.get()
 
-  if (analytics[ANALYTICS.selected] !== true) {
+  if (analytics[ANALYTICS.selected] === true) {
+    await request.cache().helpers.analytics.set({
+      [ANALYTICS.seenMessage]: true
+    })
+  } else {
```

Comment:
I think you could combine this with the following `if`, given that there are no other clauses in the block. So:
```
} else if (payload.analyticsReponse === 'accept') {
```